openshift-s2i/README.md

```diff
@@ -0,0 +1,35 @@
+# S2I
+
+This Task builds source into a container image using Project Atomic's
+[source-to-image/s2i](https://github.com/openshift/source-to-image). 
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/s2i/s2i-nodejs.yaml
+```
+
+## Inputs
+
+### Parameters
+
+* **BUILDER_IMAGE:**: The name of the image containing the s2i command. See
+  note below.  (_default:_ docker.io/kbaig/s2i)
```

Comment:
`docker.io/kbaig/s2i` -> `quay.io/redhat-pipelines/s2i`?

Reply:
@hrishin it's `quay.io/openshift-pipelines/s2i` :wink: